<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestPluginSettingsPaneClass</class>
 <widget class="QWidget" name="TestPluginSettingsPaneClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>223</width>
    <height>89</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestPluginSettingsPane</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>&amp;Name:</string>
     </property>
     <property name="buddy">
      <cstring>greetingNameInput</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="greetingNameInput">
     <property name="toolTip">
      <string>&lt;html&gt;The name of the one to greet&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="richTextCheckbox">
     <property name="toolTip">
      <string>&lt;html&gt;Use formatted text&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>&amp;Rich text</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>4</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="failCheckbox">
     <property name="toolTip">
      <string>&lt;html&gt;Cause an error when saving the settings&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>&amp;Error on save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
